src/app/zap-templates/common/override.js

```diff
@@ -15,25 +15,26 @@
  *    limitations under the License.
  */
 
-function atomicType(arg)
-{
+function atomicType(arg) {
```

Comment:
Again, why all the extraneous changes?

Reply:
Same as previous comments, something is reformatting my file when it shouldn't be.